microprofile/metrics-smallrye/src/main/java/org/wildfly/extension/microprofile/metrics/MicroProfileMetricsLogger.java

```diff
@@ -20,12 +20,13 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
 
-package org.wildfly.extension.microprofile.metrics._private;
+package org.wildfly.extension.microprofile.metrics;
```

Comment:
There's a reason for this package, although it wasn't really carried through (here or in many other places that have similar packages.)

The module.xml should have:

```
    <exports>
        <exclude path="org/wildfly/extension/microprofile/metrics/_private"/>
    </exports>
```

The gist of it is this interface must be public but the module.xml bit ^^^ makes it inaccessible outside the module.